src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -188,11 +188,13 @@ protected function failedAuthorization()
     /**
      * Get the validated data from the request.
      *
+     * @param  string|null  $key
+     * @param  mixed  $default
      * @return array
      */
-    public function validated()
+    public function validated(string $key = null, $default = null)
```

Comment:
This is unfortunately a breaking change.

Reply:
damn :(  should I PR this on master branch instead?

Reply:
a different way could be add another method `get_validated(string $key = null, $default = null)` and leave this on 8.x